.drone.yml

```diff
@@ -56,21 +56,18 @@ pipeline:
       event: [ push, tag, pull_request ]
 
   build-without-gcc:
-    image: webhippie/golang:1.8
+    image: golang:1.10
     pull: true
```

Comment:
This should be left as golang:1.8

Reply:
https://github.com/go-gitea/gitea/pull/3899/commits/e054e92104098c61caf82172592b0fbf5dd06348 Done!